NORDIC Embedded Software/nrf52 Patient Monitoring Device/main.c

```diff
@@ -106,30 +104,30 @@ void assert_nrf_callback(uint16_t line_num, const uint8_t * p_file_name)
 }
 
 
-// ALREADY_DONE_FOR_YOU: This is a timer event handler
+// This is a timer event handler
 static void timer_timeout_handler(void * p_context)
 {
-	  //Get and send data
-		int ACC ;
-		uint32_t HRS_Data = 99 ;
+    //Get and send data
+    int16_t ACC ;
+    uint32_t HRS_Data = 99 ; //default value to debug fast 
```

Comment:
This value will be updated in next lines. Do You want to set up here this value?